src/virtualenv/app_data/via_disk_folder.py

```diff
@@ -155,6 +157,14 @@ def write(self, content):
             folder.mkdir(parents=True, exist_ok=True)
         except OSError:
             pass
+        with tempfile.NamedTemporaryFile("w", prefix="atomic-write", delete=False) as tf:
```

Comment:
drop the prefix is unnecessary

Reply:
Please don't use short variable names, IMHO ``<4`` should only be used as variables within comprehensions.